.circleci/config.yml

```diff
@@ -50,6 +48,18 @@ jobs:
       - goreleaser:
           release-type: << parameters.release-type >>
 
+  acceptance_test:
+    docker:
+      - image: circleci/golang:1.14
```

Comment:
1.15 is available if that is of interrest :)

Reply:
updated